content/rdi/data-transformation/data-transformation-pipeline.md

```diff
@@ -16,6 +16,37 @@ Each job describes the transformation logic to perform on data from a single sou
 
 ![Data transformation pipeline, high level](/images/rdi/data-transformation-pipeline.png)
 
+## Default Job
+In situations where there is a need to perform a default data transformation, the default job is used. The transformation associated with this job will be applied to all tables that lack their own explicitly defined jobs. The default job must have a table name of "*", and only one instance of this type of job is permitted.
+
+For example, the default job can be used to implement common things like adding a prefix/postfix to all Redis keys or injecting a new field for each new hash or JSON written to the target without the need to define such transformations individually for each source table.
+
+At this moment the default job is supported for 'ingest' pipelines only.
+
+### Example
+This example demonstrates the process of adding an app_code field with a value of 'rdi' using the `add_field`` block to all tables that lack explicitly defined jobs. Additionally, it appends an 'aws' prefix and a 'gcp' postfix to every generated hash key.
```

Comment:
The fragment "... using the `add-field` block to..." just doesn't read well. As I'm not sure what the intent is here, can you re-word this? I did fix up some quoting stuff below.

```suggestion
This example demonstrates the process of adding an `app_code` field with a value of `rdi` using the `add_field` block to all tables that lack explicitly defined jobs. Additionally, it appends an 'aws' prefix and a 'gcp' postfix to every generated hash key.
```

Reply:
please use another value than 'rdi' in the example as it will just create confusion
use 'foo'

Reply:
Removed extra characters